src/components/dashboard/TeamMembersTable/TeamEditInfo.js

```diff
@@ -0,0 +1,121 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { useMutation } from '@apollo/client';
+import { UPDATE_TEAM_NAME, UPDATE_TEAM_ORGANIZATION } from 'data/gql/team';
+import { UPDATE_USER_TEAM_NAME, UPDATE_USER_TEAM_ORGANIZATION } from 'data/actions/type';
+import Toast from 'components/dashboard/Toast/Toast';
+
+import {
+  Box,
+  VStack,
+  Input,
+  Heading,
+  useToast,
+  Button,
+  FormControl,
+  FormLabel,
+  FormErrorMessage,
+  chakra,
+} from '@chakra-ui/react';
+
+const EditTeamInfo = ({ setOnEditPage, teamId, teamName, teamOrg }) => {
+  const toast = useToast();
+  const dispatch = useDispatch();
+  const [newTeamName, setNewTeamName] = useState('');
+  const [newTeamOrg, setNewTeamOrg] = useState('');
```

Comment:
what youd id before in initializing the state to `teamName` / `teamOrg` seemed to make more sense right?